chart/kubeapps/templates/kubeappsapis/deployment.yaml

```diff
@@ -189,15 +189,13 @@ spec:
           livenessProbe: {{- include "common.tplvalues.render" (dict "value" (omit .Values.kubeappsapis.livenessProbe "enabled") "context" $) | nindent 12 }}
             exec:
               command: ["grpc_health_probe", "-addr=:{{ .Values.kubeappsapis.containerPorts.http }}"]
-            initialDelaySeconds: 10
```

Comment:
Wondering why these initial delay seconds are removed?

Reply:
When I ran `helm template ...` I noticed we are yielding duplicated `initialDelaySeconds` keys:

![image](https://github.com/vmware-tanzu/kubeapps/assets/11535726/6e30fa8e-b9ba-4020-8de3-2ba2065aabf7)

It seems we are adding just everything we have in the values.yaml file, that is:

```yaml
  livenessProbe:
    enabled: true
    initialDelaySeconds: 60
    periodSeconds: 10
    timeoutSeconds: 5
    failureThreshold: 6
    successThreshold: 1
```

So, if we hardcode the `initialDelaySeconds: 10` in the template, we would be rendering it twice. I'd say it's better to have it in the values file rather than hardcoded in the template, no?

Reply:
Yep - definitely. I didn't notice from the above diff that we were rendering the whole map (other than the enabled key). Thanks.